src/test/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactoryUnitTest.java

```diff
@@ -1402,8 +1410,13 @@ void createNonBasicFuncotations(final int start, final int end) {
             featureList.add( gene );
             final List<Funcotation> funcotations = funcotationFactory.createFuncotationsOnVariant(variantContext, referenceContext, featureList);
 
-            // Make sure we get what we expected:
-            Assert.assertEquals(funcotations.size(), 0);
+            // Make sure we get what we expected (a single IGR funcotation):
+            Assert.assertEquals(funcotations.size(), 1);
+            Assert.assertTrue(funcotations.get(0) instanceof GencodeFuncotation);
+
+            final GencodeFuncotation gencodeFuncotation = (GencodeFuncotation)funcotations.get(0);
+            Assert.assertEquals(gencodeFuncotation.getVariantClassification(), GencodeFuncotation.VariantClassification.IGR);
+            Assert.assertNull(gencodeFuncotation.getHugoSymbol());
```

Comment:
As we discussed in person it would be beneficial to make sure `testCreateFuncotations` is run with a non-zero flanking size for 5' and 3'.

See line 1475

(Putting this here because I can't put it lower in github.)

Reply:
Turned on both 5' and 3' flanks for testCreateFuncotations(), and all test cases are passing.